src/PositionManager.sol

```diff
@@ -110,24 +110,20 @@ contract PositionManager is
     using SafeTransferLib for *;
     using CurrencyLibrary for Currency;
     using PoolIdLibrary for PoolKey;
-    using PositionConfigLibrary for PositionConfig;
     using StateLibrary for IPoolManager;
     using TransientStateLibrary for IPoolManager;
     using SafeCast for uint256;
     using SafeCast for int256;
     using CalldataDecoder for bytes;
     using SlippageCheckLibrary for BalanceDelta;
-    using PositionConfigIdLibrary for PositionConfigId;
+    using PoolKeyChecker for PoolKey;
+    using PositionInfoLibrary for PackedPositionInfo;
```

Comment:
nit: would rather the names were the same like
`PositionInfo` and `PositionInfoLibrary`
or `PackedPositionInfo` and `PackedPositionInfoLibrary`

Reply:
I prefer the first one i think. Packing it is just an implementation detail - so `PositionInfo` feels better

Reply:
sgtm!